pkg/asset/templates/kubelet-master.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kubelet
+  namespace: kube-system
+  labels:
+    k8s-app: kubelet
+    version: v1.2.4_inotify.3
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: kubelet
+        version: v1.2.4_inotify.3
+    spec:
+      nodeSelector:
+        master: "true"
+      containers:
+      - name: kubelet
+        image: quay.io/peanutbutter/hyperkube:v1.2.4_inotify.3
+        command:
+        - /nsenter
+        - --target=1
+        - --mount
+        - --wd=.
+        - --
+        - ./hyperkube
+        - kubelet
+        - --api-servers={{ index .APIServers 0 }}
+        - --config=/etc/kubernetes/manifests
+        - --allow-privileged
+        - --hostname-override=$(MY_POD_IP)
+        - --cluster-dns=10.3.0.10
+        - --cluster-domain=cluster.local
+        - --kubeconfig=/etc/kubernetes/kubeconfig
+        - --lock-file=/var/run/lock/kubelet.lock
+        - --node-labels=master=true
```

Comment:
Ah, so a kubelet coming up on the same host _without_ that flag will not negate it?